Shared/Samples/Show viewshed from geoelement in scene/README.md

```diff
@@ -0,0 +1,47 @@
+# Show viewshed from geoelement in scene
+
+Analyze the viewshed for an object (GeoElement) in a scene.
```

Comment:
Instead of using the API name in description, let's use the plain English from the [terminology guide](https://csreview.esri.com/terminology-guide)
```suggestion
Analyze the viewshed for an object (geoelement) in a scene.
```